cmd/tss/main.go

```diff
@@ -67,6 +62,7 @@ func main() {
 		tssConf,
 		nil,
 		p2pConf.ExternalIP,
+		os.Getenv("PASSWORD"), //TODO: password missing?
```

Comment:
> not sure what to do with password, this was just broken because argument was not provided, i assume setting env is good enough, we are not using this part of the code anyways

From the upgrade it became broken?

@brewmaster012 any idea on this?